.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "libdbus-sys/vendor/dbus"]
+	path = libdbus-sys/vendor/dbus
+	url = https://github.com/freedesktop/dbus.git
```

Comment:
Would it be possible to pin this to a specific tag in the repository to reduce the risk of the vendoring build breaking when the dbus.git repo changes?

Reply:
The submodule is actually pinned to the following git tag/release:

```
$ git submodule status
 ed866a94889e13c83dc873d8b5f86a907f908456 libdbus-sys/vendor/dbus (dbus-1.15.2)
 ```
 
 For some reason git doesn't show that in the `.gitmodules` file, only in the git object tree itself.

Reply:
I looked it up; specifying a sha or tag is not supported, so for stability I suggest we go with the 1.14 branch. You can specify `branch = 1.14` in .gitmodules. When the 1.16 branch is stable I'll be happy to take a PR for that, but the 1.15 branch is a development branch which is more likely to break from time to time, both w r t build failures and runtime bugs.

Does that make sense?

Reply:
The git-submodule is already tagged at their 1.15.2 release though. If you clone this branch of `dbus-rs` you'll be pinned at `dbus-1.15.2`.

And when we publish to crates.io it will use that tag. Similar to other sys crates, like https://github.com/alexcrichton/openssl-src-rs.

Current tag: https://github.com/freedesktop/dbus/releases/tag/dbus-1.15.2

So I don't think we need to pin to a development branch.

Reply:
Oh I see what you mean, 1.15 is a development release. Yeah we can pin to the stable 1.14 instead.

Reply:
Okay now pinned at 1.14.4. Completed in dbc134d22e78aa47cd9f7bb62c4c2e1387c2150f and 4b252a8ce0831ade5473f77b7271aa83c2400ec2.